roles/core/templates/sdcore-5g-values.yaml

```diff
@@ -323,20 +325,23 @@ omec-user-plane:
         subnet: {{ ran_subnet }} # Subnet for the gNB network
       access:
         iface: {{ core.data_iface }}
+        resourceName: "intel.com/intel_sriov_vfio_access"
         gateway: 192.168.252.1
         ip: 192.168.252.3/24
       core:
         iface: {{ core.data_iface }}
+        resourceName: "intel.com/intel_sriov_vfio_core"
         gateway: 192.168.250.1
         ip: 192.168.250.3/24
       cfgFiles:
         upf.jsonc:
-          mode: af_packet                # This mode implies no DPDK
+          mode: {{ core.upf.mode }}
           hwcksum: true
           log_level: "trace"
           measure_upf: true
           measure_flow: true
           gtppsc: true                   # Extension header enabled in 5G.
+          ddp: {{ core.upf.ddp }}
```

Comment:
Should we add a comment here indicating that this is an Intel specific feature/capability?

Reply:
done